openstates/ky/bills.py

```diff
@@ -40,7 +40,7 @@ def scrape_subjects(self, session):
                 for bill in sdoc.xpath('//div[@id="bul"]/a/text()'):
                     self._subjects[bill.replace(' ', '')].append(subject)
 
-    def scrape(self, session=None, chamber=None):
+    def scrape(self, session=None, chamber=None, prefile=None):
```

Comment:
If `prefile` is a flag, I would expect the default to be a boolean, probably `False`.

Reply:
Good catch, i'll make this change later.